sapi/apache2handler/sapi_apache2.c

```diff
@@ -656,6 +665,13 @@ zend_first_try {
 		brigade = ctx->brigade;
 	}
 
+	if (SG(request_info).content_length > SG(read_post_bytes)) {
+		ap_log_rerror(APLOG_MARK, APLOG_ERR, 0, r, "attemp to read POST data error: %d", SG(sapi_headers).http_response_code);
```

Comment:
Typo in error message: attemp -> attempt? Though maybe something like "Error while attempting to read POST data: %d" might be better.